frontend/src/app/filter/range-filter/range-filter.component.html

```diff
@@ -1,6 +1,8 @@
-<div class="range" *ngIf="(filter?.state$ | async).data as data">
-    <span>{{data.min}} - {{data.max}}</span>
-    <p-slider [animate]="true" [ngModel]="[data.min, data.max]"
-        [range]="true" [min]="min" [max]="max"
-        (onChange)="sliderValue$.next($event.values)"></p-slider>
-</div>
+<ng-container *ngIf="min !== undefined && max !== undefined">
```

Comment:
I presume the `*ngIf` is to delay loading the component until after the `min` and `max` are set?
In that case we need to be sure that these are always set.

Reply:
Yes, you can't really make a slider otherwise. The same component _should_ be requesting a min/max value.